ui/src/components/common/TableView.js

```diff
@@ -1,21 +1,21 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Table, TableHeader, TableBody } from '@patternfly/react-table';
-import { withRouter } from 'react-router-dom';
+import './TableView.css';
 
-export const AppsTable = ({ columns, rows, sortBy, onSort, onRowClick }) => (
+export const TableView = ({ columns, rows, sortBy, onSort, onRowClick }) => (
```

Comment:
We have an `AppView` and a `TableView` - we should probably use different terminology for these different component types. It could be `AppPage` and `Table` or `AppView` and `Table`.